.eslintrc.json

```diff
@@ -28,12 +30,34 @@
     "ecmaVersion": 2019,
     "sourceType": "module"
   },
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  },
   "rules": {
     "no-console": "warn",
     "no-debugger": "error",
-    "import/order": "warn",
+    "sort-imports": "off",
+    "import/order": "off",
     "import/export": "off",
+    "import/first": "error",
+    "import/newline-after-import": "error",
     "import/no-extraneous-dependencies": "error",
+    "import/no-duplicates": "error",
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          ["^\\u0000", "/(fixtures|mocks|patches)$"],
+          ["^@?\\w"],
+          ["^raiden-\\w+"],
+          ["^"],
```

Comment:
If this should be the pattern for absolute paths, why not adding the leading slash? Because this way it looks a bit awkward/information less.

Reply:
https://github.com/lydell/eslint-plugin-simple-import-sort#custom-grouping , this is just a customization of the `default groups`. Information-less is mostly because I had to remove the comments for JSON.

Reply:
Alright. According to the comments below I misunderstood the meaning/purpose of this pattern.